starter-code/views/index.ejs

```diff
@@ -1,10 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>The Chuck Norris Joke Department</title>
-    <link rel="stylesheet" type="css/text" href="../public/stylesheets/style.css">
-  </head>
-  <body>
 
-  </body>
+<head>
+  <title>The Chuck Norris Joke Department</title>
+  <link rel="stylesheet" type="css/text" href="../public/stylesheets/style.css">
+</head>
+
+<body>
+  <%- body -%>
```

Comment:
You are puttin <%- body -%> here and in main-layout. I assume you only wanted to put it in main-layout.